packages/service/lib/schema.js

```diff
@@ -548,13 +548,20 @@ const metrics = {
     {
       type: 'object',
       properties: {
-        port: {
+        server: {
```

Comment:
Is there a changelog that should list config schema changes?

Reply:
You need to keep supporting the old option too, we are not planning a breaking change anytime soon.

Reply:
I refactored to avoid changing the schema for port and hostname.